core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -957,6 +988,20 @@ private String remove0(int h, int i, IN_NAME name) {
 
     // Conversion functions
 
+    @Nullable
+    private static Boolean toBoolean(@Nullable String v) {
+        if (v == null) {
+            return null;
+        }
+        if ("true".equalsIgnoreCase(v)) {
+            return true;
+        }
+        if ("false".equalsIgnoreCase(v)) {
+            return false;
+        }
```

Comment:
Should we convert a number to a boolean, like 0 to false and other to true? What do we do for other values like on/off and yes/no? What's the widely accepted sane behavior among other frameworks?

Reply:
Shouldn't we handle `0` and `1` for consistency with annotated services?

Reply:
Yeah, I will do. Just waiting for the result of https://github.com/line/armeria/issues/3619 and I think I can do it now. 😄

Reply:
Fixed. 😉